components/builder-web/app/origin/origin-page/origin-jobs-tab/_origin-jobs-tab.component.scss

```diff
@@ -0,0 +1,8 @@
+.origin-jobs-tab-component {
+  display: flex;
+  flex-flow: column;
```

Comment:
Why use flexbox here and not just `display: block`, since these children are just stacked and there's no call for column wrapping?

Reply:
`display: block` didn't work :)

Reply:
Happy to pair with you on this, then; I can't think of a reason good ol' blocks wouldn't work here. If they don't, something else must be going on.

Reply:
So let's remove this file altogether, then, since the flexbox isn't needed, and use a `section` to contain the list.